flink-ml-lib/src/test/java/org/apache/flink/ml/feature/CountVectorizerTest.java

```diff
@@ -0,0 +1,391 @@
+                                    IntStream.of(3, 5).toArray(),
+                                    DoubleStream.of(1.0, 1.0).toArray()),
+                            Vectors.sparse(
+                                    6,
+                                    IntStream.of(0, 1).toArray(),
+                                    DoubleStream.of(1.0, 2.0).toArray())));
+
+    @Before
+    public void before() {
+        Configuration config = new Configuration();
+        config.set(ExecutionCheckpointingOptions.ENABLE_CHECKPOINTS_AFTER_TASKS_FINISH, true);
+        env = StreamExecutionEnvironment.getExecutionEnvironment(config);
+        env.setParallelism(4);
+        env.enableCheckpointing(100);
+        env.setRestartStrategy(RestartStrategies.noRestart());
+        tEnv = StreamTableEnvironment.create(env);
+
+        trainDataTable = tEnv.fromDataStream(env.fromCollection(TRAIN_DATA)).as("input");
+    }
+
+    private static void verifyPredictionResult(
+            Table output, String outputCol, List<SparseVector> expected) throws Exception {
+        StreamTableEnvironment tEnv =
+                (StreamTableEnvironment) ((TableImpl) output).getTableEnvironment();
+        DataStream<SparseVector> stream =
+                tEnv.toDataStream(output)
+                        .map(
+                                (MapFunction<Row, SparseVector>)
+                                        row -> (SparseVector) row.getField(outputCol));
+        List<SparseVector> result = IteratorUtils.toList(stream.executeAndCollect());
+        compareResultCollections(expected, result, TestUtils::compare);
+    }
+
+    @Test
+    public void testParam() {
+        CountVectorizer countVectorizer = new CountVectorizer();
+        assertEquals("input", countVectorizer.getInputCol());
+        assertEquals("output", countVectorizer.getOutputCol());
+        assertEquals((double) Long.MAX_VALUE, countVectorizer.getMaxDF(), EPS);
+        assertEquals(1.0, countVectorizer.getMinDF(), EPS);
+        assertEquals(1.0, countVectorizer.getMinTF(), EPS);
+        assertEquals(1 << 18, countVectorizer.getVocabularySize());
+        assertFalse(countVectorizer.getBinary());
+
+        countVectorizer
+                .setInputCol("test_input")
+                .setOutputCol("test_output")
+                .setMinDF(0.1)
+                .setMaxDF(0.9)
+                .setMinTF(10)
+                .setVocabularySize(1000)
+                .setBinary(true);
+        assertEquals("test_input", countVectorizer.getInputCol());
+        assertEquals("test_output", countVectorizer.getOutputCol());
+        assertEquals(0.9, countVectorizer.getMaxDF(), EPS);
+        assertEquals(0.1, countVectorizer.getMinDF(), EPS);
+        assertEquals(10, countVectorizer.getMinTF(), EPS);
+        assertEquals(1000, countVectorizer.getVocabularySize());
+        assertTrue(countVectorizer.getBinary());
+    }
+
+    @Test
+    public void testInvalidParam() {
```

Comment:
This test case only tests `minDF` and `maxDF`, not generally all parameters. We can modify the name of this test case to reflect this, or add tests for other possibly invalid parameters.